chainer/functions/connection/deconvolution_nd.py

```diff
@@ -326,54 +326,99 @@ def deconvolution_nd(x, W, b=None, stride=1, pad=0, outsize=None,
     """N-dimensional deconvolution function.
 
     This is an implementation of N-dimensional deconvolution which generalizes
-    two-dimensional one. It takes three variables: input ``x``, the filter
-    weight ``W``, and the bias vector ``b``.
+    two-dimensional one. In most of deep learning frameworks and papers, this
+    function is called **transposed convolution**. But because of historical
+    reasons (e.g. paper by Ziller `Deconvolutional Networks`_) and backward
+    compatibility, this function is called **deconvolution** in Chainer.
+
+    .. _Deconvolutional Networks: \
+http://www.matthewzeiler.com/pubs/cvpr2010/cvpr2010.pdf
+
+    It takes three variables: the input ``x``, the filter weight ``W``, and the
+    bias vector ``b``.
+
+    Notation: here is a notation for dimensionalities.
+
+    - :math:`N` is the number of spatial dimensions.
+    - :math:`n` is the batch size.
+    - :math:`c_I` and :math:`c_O` are the number of the input and output
+      channels, respectively.
+    - :math:`d_1, d_2, ..., d_N` are the size of each axis of the input's
+      spatial dimensions, respectively.
+    - :math:`k_1, k_2, ..., k_N` are the size of each axis of the filters,
+      respectively.
+    - :math:`l_1, l_2, ..., l_N` are the size of each axis of the output's
+      spatial dimensions, respectively.
```

Comment:
I think it is easier to read if we remove the definition of `l` from the list because `l` depends on other items in the list and we can highlight that `l` is determined by the items in the list.